robyn/router.py

```diff
@@ -290,10 +290,15 @@ def add_auth_middleware(self, endpoint: str):
 
         def decorator(handler):
             @wraps(handler)
-            def inner_handler(request: Request, *args):
+            async def inner_handler(request: Request, *args):
```

Comment:
and what about sync support?

Reply:
I tested with both sync and async and both worked pretty well...
However, I'm kind of worried not showing  support for both sync and async. I'll make some tests and will back with the results